### ISSUE TITLE
413: Request Entity Too Large 

### ISSUE THREAD

**Issue by ychhajed** [jsreport/jsreport#257]
We have our standalone jsreport server installation on aws. We call the API to get the jsreport from a .net application. The problem we are facing is when the JSON data is very big and we are generating 100's of reports in one pdf we get the below error:
Unable to render template.  
413 Request Entity Too Large
nginx

We know, we need to change the limits w=either on the nginx or in the jsreport configuration. We tried both but both are not working for us.

Can you please help what we are missing here. How shall we update the elastic beanstalk setting for this to take affect ?

function call used from .Net application is below : 
_reportingService.RenderAsync(jsreportId, jsonData);

below is the jsreport server files we deploy on elastic beanstalk instance. 
[jsreport.zip](https://github.com/jsreport/jsreport/files/697358/jsreport.zip)

Thanks

**Comment by pofider** [comment]
Your jsreport configuration is correct. Just make sure you run in development mode - check the tab title in the browser. If you run in production mode, you should create `prod.config.json` file.

The configuration of elastic beanstalk and nginx is a different question. I too busy now to play with beanstalk, but here seems to be solution
http://stackoverflow.com/questions/18908426/increasing-client-max-body-size-in-nginx-conf-on-aws-elastic-beanstalk
Please make a note here if it works.

**Comment by ychhajed** [comment]
Thanks. We did try this again on our DEV instance which does not have load balancer and it worked. NGINX engine was restarted using the command in the file and it worked on DEV. The task to restart nginx did not work on production which had a load balancer. We had to increase the load balancer timeout also.

We have it working in both DEV & PRD but we are not able to still confirm whether we need to restart nginx engine after the deployment. (We will not try this now...but may be next time we get into this we will....)
